apps/wallet-ui/src/app/[ecosystem]/(ui)/wallet/[address]/page.tsx

```diff
@@ -0,0 +1,17 @@
+import NftGallery, { NftGalleryLoading } from "@/components/NftGallery";
+import { Suspense } from "react";
+import { isAddress } from "thirdweb";
+
+export default async function Page({
+  params: { address },
+  searchParams: { chainId },
+}: {
+  params: { address: string };
+  searchParams: { chainId: string };
+}) {
+  return (
+    <Suspense fallback={<NftGalleryLoading />}>
```

Comment:
remind me why this couldn't just be:
`loading.tsx` -> gallery loading component
`page.tsx` -> gallery component?